src/Shouldly.Tests/ShouldNotContain/StringContainsStringCaseIsInsensitiveScenario.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.ShouldNotContain
+{
+    public class StringContainsStringCaseIsInsensitiveScenario : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Shouldly is legendary".ShouldNotContain("legendary", Case.Insensitive);
```

Comment:
I'm having a bit of difficulty in creating the ShouldNotContainCaseInsensitiveMessageGenerator. Within the `CanProcess` message I need to check the assertion context to see if the method signature is either Case.Sensitive or Case.Insensitive; however it doesn't look like this can be reflected from `MethodBase` returned by UnderlyingShouldMethod (presuming this is how I'm supposed to be going about it).

Do you have any suggestions?